ProjFS.Linux/PrjFSLib.Linux.Managed/VirtualizationInstance.cs

```diff
@@ -26,25 +28,35 @@ public static Result ConvertDirectoryToVirtualizationRoot(string fullPath)
         }
 
         public virtual Result StartVirtualizationInstance(
+            string storageRootFullPath,
             string virtualizationRootFullPath,
             uint poolThreadCount)
         {
+            if (mountHandle != IntPtr.Zero)
+            {
+                throw new InvalidOperationException();
+            }
+
             Interop.Callbacks callbacks = new Interop.Callbacks
             {
                 OnEnumerateDirectory = this.OnEnumerateDirectory,
                 OnGetFileStream = this.OnGetFileStream,
                 OnNotifyOperation = this.preventGCOnNotifyOperationDelegate = new NotifyOperationCallback(this.OnNotifyOperation),
             };
 
-            return Interop.PrjFSLib.StartVirtualizationInstance(
+            Result result = Interop.PrjFSLib.StartVirtualizationInstance(
+                storageRootFullPath,
                 virtualizationRootFullPath,
                 callbacks,
-                poolThreadCount);
+                poolThreadCount,
+                ref this.mountHandle);
+
+            return result;
         }
 
-        public virtual Result StopVirtualizationInstance()
+        public virtual void StopVirtualizationInstance()
         {
-            throw new NotImplementedException();
+            Interop.PrjFSLib.StopVirtualizationInstance(this.mountHandle);
```

Comment:
Minor: clear out `this.mountHandle`

Reply:
Good catch!  The latest commit 81f4923 should do this now.